### PR TITLE
Fixing tests

### PR DESCRIPTION
**Pull request by andrewram4287**

### CODE REVIEW

File: gmm/figures/figure1.py [hunk 1]
```diff
@@ -18,30 +19,32 @@ def makeFigure():
...
+    # ax[5].axis("off")
 
     # Add subplot labels
     subplotLabel(ax)
 
     # smallDF(Amount of cells wanted per experiment); 336 conditions in total
     cellperexp = 200
-    zflowDF, experimentalcells = smallDF(cellperexp)
+    uniq = 336
+    zflowDF, experimentalcells, concDF, ligDF, valDF, timeDF = smallDF(cellperexp)
```
Comment (aarmey) [inline gmm/figures/figure1.py:30]: Why not just keep these variables in the dataframe, and then remove them later when they shouldn't be around (e.g., in the GMM)?

File: gmm/figures/figure1.py [hunk 2]
```diff
@@ -52,10 +55,49 @@ def makeFigure():
...
+
+    
+    for i,lig in enumerate(np.unique(ligDF)):
+        for j,tim in enumerate(np.unique(timeDF)):
+            for k in range(maxcluster):
+    
+                specificDF = statDF.loc[(statDF["Ligand"] == lig) & (statDF["Time"] == tim) &
+                             (statDF["Cluster"] == k) & (statDF["Valency"] == 1)]
+
+                ax[4].scatter(specificDF["Concentration"].values,specificDF["pSTAT5"].values,label = k)
```
Comment (aarmey) [inline gmm/figures/figure1.py:87]: Look at https://seaborn.pydata.org/generated/seaborn.scatterplot.html

It won't require the loops.

File: gmm/figures/figure1.py [hunk 3]
```diff
@@ -52,10 +55,49 @@ def makeFigure():
...
+            
+            if j == 0:
```
Comment (aarmey) [inline gmm/figures/figure1.py:89]: What are the breaks meant to do?

Comment (andrewram4287) [reply gmm/figures/figure1.py:89]: I only wanted to plot one graph for one specific mutein, one valency, and time point. I was able to create the large DF, but now just looking at pSTAT5 vs. concentration.

**Comment by aarmey** [comment]
Found a bug of my own doing. I added normalization of pSTAT5 _within each condition_. This was removing any variation of pSTAT5 over time, concentration, ligand, etc. Fixed in last commit.

**Pull request closed without merging**